tests/security/selinux/sestatus.pm

```diff
@@ -34,6 +34,9 @@ sub run {
     # SELinux by default
     validate_script_output("sestatus", sub { m/SELinux status: .*disabled/ });
 
+    # workaround for "selinux-auto-relabel" in case: auto relabel then trigger reboot
```

Comment:
Just have a quick question about GRUB_TIMEOUT=8 usage here.

Reply:
> Just have a quick question about GRUB_TIMEOUT=8 usage here.

Answered you offline.